.github/cspell.json

```
@@ -0,0 +1,18 @@
+{
+  "version": "0.2",
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "dictionaries": ["vgv_allowed", "vgv_forbidden"],
+  "dictionaryDefinitions": [
+    {
+      "name": "vgv_allowed",
```

```suggestion
      "name": "very_allowed_words",
```

Joking, but would be cool :P